NAMESPACE

```diff
@@ -84,5 +87,7 @@ importFrom(data.table,fread)
 importFrom(dplyr,"%>%")
 importFrom(lazyeval,interp)
 importFrom(lazyeval,lazy)
+importFrom(limma,squeezeVar)
```

Comment:
is this used anywhere? I don't see it

Reply:
just kidding, see it now. hm. I'd like to not depend on yet another package, ideally